app/store/services/activities.js

```diff
@@ -95,36 +95,76 @@ export const getAthleteActivities = token => {
   );
 };
 
-export const getRankings = (activities = {}) => {
+export const getRankings = (members = [], activities = {}) => {
```

Comment:
i think the body of this function can be done in a smarter way but is it worth the work?